packages/webpack/src/builder.js

```diff
@@ -0,0 +1,284 @@
+import path from 'path'
+import fs from 'fs'
+import pify from 'pify'
+import map from 'lodash/map'
+import debounce from 'lodash/debounce'
+import concat from 'lodash/concat'
+import omit from 'lodash/omit'
+import uniq from 'lodash/uniq'
+import values from 'lodash/values'
+import chokidar from 'chokidar'
+import webpack from 'webpack'
+import MFS from 'memory-fs'
+import webpackDevMiddleware from 'webpack-dev-middleware'
+import webpackHotMiddleware from 'webpack-hot-middleware'
+import upath from 'upath'
+import consola from 'consola'
+
+import {
+  r,
+  parallel,
+  sequence
+} from '@nuxt/common'
+
+import { ClientConfig, ServerConfig, PerfLoader } from './config'
+
+export default class WebpackBuilder {
+  constructor(context) {
+    this.context = context
```

Comment:
For easier tracing and allowing context references to be modified on runtime, suggest just keep using `this.context` 👍

Reply:
Cool